lms/djangoapps/courseware/views.py

```diff
@@ -868,6 +869,7 @@ def course_about(request, course_id):
     with modulestore().bulk_operations(course_key):
         permission = get_permission_for_course_about()
         course = get_course_with_access(request.user, permission, course_key)
+        course_details = CourseDetails.fetch(course_key)
```

Comment:
We should optimize this by adding a `@classmethod` to `CourseDetails` that takes the course descriptor fetched on the line above as an argument and uses it to populate the `CourseDetails` object instead of doing a second course lookup.